docker/Dockerfile.build-linux

```diff
@@ -30,4 +30,4 @@ ENV GOPATH /go
 RUN mkdir -p /go
 WORKDIR /build
 
-RUN curl -LO https://storage.googleapis.com/lp_testharness_assets/tasmodel.pb
\ No newline at end of file
+RUN curl -LO https://github.com/livepeer/livepeer-ml/releases/download/v0.1/tasmodel.pb
```

Comment:
Is there no functional difference between v0.1 and v0.2/v0.3 listed in https://github.com/livepeer/livepeer-ml/releases ?

Reply:
None, it was test runs to make sure the runner is fully functional. I'll change it to point to 0.3.